rental_main.rb

```diff
@@ -0,0 +1,28 @@
+require_relative './rental'
+
+class Rentalmain
+  def create_a_rental(books, people, rental)
+    puts 'Select a book from the following list of books'
+    books.each_with_index { |n, idx| puts "#{idx}) #{n.to_string}" }
+    var = gets.chomp
+    book = books[var.to_i]
+    puts 'Select a person from the following list by number'
+    people.each_with_index { |n, idx| puts "#{idx}) #{n.to_string}" }
+    vars = gets.chomp
+    person = people[vars.to_i]
+    puts 'Enter a date'
+    date = gets.chomp
```

Comment:
- [ ] Kindly ensure that these requests for user inputs are refactored into a different method.

Reply:
Thank you for your feedback, Changes will be implemented